File: app/src/main/java/com/cpiekarski/fourteeners/activities/LeaveNoTraceActivity.java
package com.cpiekarski.fourteeners.activities;

import android.app.Activity;
import android.os.Bundle;

import com.cpiekarski.fourteeners.R;


public class LeaveNoTraceActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_leave_no_trace);
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
    }
}

File: app/src/main/kotlin/com/cpiekarski/fourteeners/activities/LeaveNoTraceActivity.kt
package com.cpiekarski.fourteeners.activities

import android.app.Activity
import android.os.Bundle
import com.cpiekarski.fourteeners.R

class LeaveNoTraceActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_leave_no_trace)
    }

    override fun onDestroy() {
        super.onDestroy()
    }
}